doc/src/stdlib/numbers.md

```diff
@@ -2,22 +2,39 @@
 
 ## Standard Numeric Types
 
-  * `Bool`
-  * `Int8`
-  * `UInt8`
-  * `Int16`
-  * `UInt16`
-  * `Int32`
-  * `UInt32`
-  * `Int64`
-  * `UInt64`
-  * `Int128`
-  * `UInt128`
-  * `Float16`
-  * `Float32`
-  * `Float64`
-  * `Complex64`
-  * `Complex128`
```

Comment:
Perhaps. Currently `?Complex128` gives the `Complex` docstring. We could mention the aliases there? Feels weird to give aliases their own docstrings.
There is apparently also a `Complex32`